### ISSUE TITLE
Evaluation with Provided Checkpoint Only Reaches 15.64% Accuracy

### ISSUE THREAD

**Issue by Aesop-programmer** [liuzechun/ReActNet#38]
Hi, I tried to evaluate the model using the checkpoint you provided, but I can only get 15.64% top-1 accuracy on ImageNet validation.

Below is the code I used.
The part that removes the module. prefix is because I want to load the checkpoint on a single GPU.
I would like to ask whether my understanding is incorrect, or if there is something wrong in my loading/evaluation code.
```
import torch
from birealnet import birealnet18
from torchvision import datasets, transforms

model = birealnet18()
checkpoint = torch.load('path to your provided checkpoint')

# Remove 'module.' prefix from keys
new_state_dict = {
    k[len("module."):] if k.startswith("module.") else k: v
    for k, v in checkpoint["state_dict"].items()
}

model.load_state_dict(new_state_dict, strict=False)

val_transform = transforms.Compose([
    transforms.Resize(256),
    transforms.CenterCrop(224),
    transforms.ToTensor(),
    # Official normalization
    transforms.Normalize(mean=[0.485, 0.456, 0.406],
                         std=[0.229, 0.224, 0.225]),
])

imagenet_val = datasets.ImageFolder(
    root="/Imagenet2012/val",
    transform=val_transform
)

Dataloader = torch.utils.data.DataLoader(imagenet_val, batch_size=32, shuffle=False)

num = 0
correct = 0
model = model.to("cuda")

for batch in Dataloader:
    inputs, targets = batch
    inputs = inputs.to("cuda")
    targets = targets.to("cuda")

    pred = model(inputs)
    _, predicted = pred.max(1)

    correct += predicted.eq(targets).sum().item()
    num += inputs.size(0)

print(f"Accuracy: {correct / num * 100:.2f}%")
```
If you have any suggestions or if I missed any important detail in the evaluation setting, please let me know. Thanks!
